interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -773,7 +773,7 @@ namespace cling {
     CO.CodeGeneration = m_IncrParser->hasCodeGenerator();
     CO.DynamicScoping = isDynamicLookupEnabled();
     CO.Debug = isPrintingDebug();
-    CO.IgnorePromptDiags = !isRawInputEnabled();
+    CO.IgnorePromptDiags = 0;
```

Comment:
This makes things stricter, right? ie now it won't ignore prompt diagnostics even outside raw input mode by default

Reply:
Exactly :+1:.  Thanks for the review!